en/micro-integrator/docs/develop/creating-artifacts/using_docker_secrets.md

```diff
@@ -0,0 +1,100 @@
+# Using Docker Secrets
+
+When you deploy your integration solutions in a container environment, you cannot [use the Cipher Tool to encrypt secrets](../../../develop/creating-artifacts/encrypting-synapse-passwords) as you do for a VM deployment.
```

Comment:
It's not that they can't. Its more like an additional capability. 
'WSO2 Micro-integrator comes with a built-in secret repository as a part of its secure vault implementation by default. In addition to this, Micro-integrator also provides built in support for docker secrets and kubernetes secrets for your containerized deployments.
'